config/themePluginConfig.ts

```diff
@@ -0,0 +1,115 @@
+export default {
+  theme: [
+    {
+      key: 'dark',
+      fileName: 'dark.css',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'fileName' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=3667353)